assemblies/assembly-install-rhdh-ocp.adoc

```diff
@@ -5,19 +5,33 @@
 
 toc::[]
 
-You can install {product} on {ocp-short} by using one of the following methods:
+You can install {product} on {ocp-short} by using one of the following installers:
 
-* The Helm chart
 * The {product} Operator
+* The Helm chart
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc[leveloffset=+2]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+== Installing {product} on {ocp-short} by using the operator
+
+You can install {product} on {ocp-short} by using the operator with one of the following methods:
 
-// Operator method
-include::modules/installation/proc-install-rhdh-ocp-operator.adoc[leveloffset=+1]
+* The {ocp-short} console
+* The {ocp-short} CLI
```

Comment:
How do the OCP docs refer to this? Do they say something like `oc` CLI tool? We should copy whatever they do.

Reply:
The OpenShift CLI (`oc`)